changelog/hotfix-revert-460-wp-wc-min-supported-version-bump

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: dev
+
+Revert bumping minimum required version of WooCommerce to 7.8.0 and WP to 6.0
```

Comment:
I think it can be
```suggestion
Revert - Bump minimum required version of WooCommerce to 8.0.0 and WP to 6.1
Setting the minimum required version of WooCommerce back to 7.8.0 and WP to 6.0
```

Reply:
Nice. I wasn't sure about this too. I'll create a separate changelog entries because if I remember correctly, new lines in one changelog file will cause a problem when generating the final changelog.

Reply:
Addressed in [70af0ee](https://github.com/Automattic/woocommerce-payments/pull/7142/commits/70af0eed49f6d292fac8adb7db5b8b3a6a0d6bb3).